test/Worker.Extensions.Tests/Blob/StreamTests.cs

```diff
@@ -246,7 +246,7 @@ public async Task ConvertAsync_StreamCollection_WithFilePath_Throws_ReturnsFaile
             // Assert
             Assert.Equal(ConversionStatus.Failed, conversionResult.Status);
             Assert.IsType<NotSupportedException>(conversionResult.Error);
-            Assert.Contains("Deserialization of types without a parameterless constructor, a singular parameterized constructor, or a parameterized constructor annotated with 'JsonConstructorAttribute' is not supported.", conversionResult.Error.Message);
+            Assert.Contains("Deserialization of interface or abstract types is not supported.", conversionResult.Error.Message);
```

Comment:
The exception messae thrown has changed between net8.0 and net9.0+. Minimal repro to show here: https://gist.github.com/kshyju/04bd2c8d2891bd8df8b2e0d711276984#file-stjserializationerrornet8vsnet9-cs